docs/source/gbd2017_models/risks/smoking/index.rst

```diff
@@ -4,58 +4,170 @@
 
 Years Since Quitting for Former Smokers
 +++++++++++++++++++++++++++++++++++++++
 
-This is a *continuous* risk exposure model that follows an *ensemble* distribution.
+This is a *continuous* risk exposure model that follows an *ensemble* distribution that has been converted to a **categorical** risk exposure model for the CSU forecasts from 2020-2040.
+
+The categorical version of this risk exposure is binned into the following categories of pack-years: 0 to 2, 2 to 4, 5 to 9, 10 to 19, 20 to 29, 30 to 39, 40 to 49, 50 to 99.
+
+This risk exposure was forecast from 2020-2040 based on the GBD 2019 risk exposure estimates as a measure of number of individuals in each exposure category per person year in the *general population* (note that the denominator here is not person time among former smokers).
 
 Vivarium Modeling Strategy
 --------------------------
 
+The overall scope of the Vivarium modeling strategy for the smoking risk exposure for the :ref:`Lung Cancer Screening Model <lung_cancer_cancer_concept_model>` is that simulants will be assigned exposure values based on the exposure distribution for a given age-, sex-, location-, and year-specific demographic group and their exposure will be updated each time they move through these groups. Each simulant will be given a propensity score that determines the relative magnitude of their individual exposure value to the exposure distribution. However, this strategy does allow for the possibility of illogical smoking exposure transitions for individual simulants (i.e.: moving from a current smoker to a never smoker, decreases in pack-year exposure values over time).
+
+.. todo::
+
+  Quantify these possibilities based on exposure forecasts using Abie's nano sim for stomach cancer and/or Yongquans cohort plots
+
+This strategy is in contrast to modeling a rate of smoking initiation/cessation and number of cigarettes smoked, which would allow for detailed logical sample histories of each individual simulant.
+
+The below diagram displays the exposure possibilities within the Vivarium modeling strategy for the smoking risk exposure: simulants can transition between never, curent, and former smoking status groups (although the transition from current to never smokers is illogical). Additionally, current and former smokers are assigned pack-year exposures and former smokers are assigned years since quitting exposures.
+
+.. image:: smoking_exposure_diagram.svg
+
+Smoking Status
+++++++++++++++
+
+Each simulant should get a smoking status propensity score (smoking_status_propensity_i) that is a random value assigned based on a uniform distribution from 0 to 1. This propensity score for an individual should remain constant for the entire duration of the simulation. 
+
```

Comment:
The propensity score can change as simulant's smoking status changes, i assume constant number is used for initialization.

Reply:
This propensity is for smoking status only, and not for pack-years or years since quitting; propensities for each of these variables should all be separate and independent. I will add some clarification here, thanks.

Reply:
Was this what you meant?